src/targets/chrome/get-selector-box-size.js

```diff
@@ -1,10 +1,42 @@
 const getSelectorBoxSize = (window, selector) => {
-  const element = window.document.querySelector(selector);
-  if (!element) {
-    throw new Error('Unable to find element');
+  const elements = [...window.document.querySelectorAll(selector)];
+
+  if (elements.length === 0) {
+    throw new Error('Unable to find selector');
   }
-  const { x, y, width, height } = element.getBoundingClientRect();
-  return { x, y, width, height };
+
+  const removeWrapperElements = element => {
```

Comment:
This name is confusing, maybe `isNotWrapperElement` or `doesNotContainOtherElements`?

Reply:
Done! ✅